Bottom: f4277ac0e5d458e51392bc6274489f54feed4c00
Top:    d36276ef78a75a467df835da9e19cb140f565f8c
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-07-18 20:13:15 -0400

Refresh of tyler

---

diff --git a/arch/arm/boot/dts/sun7i-a20-cubieboard2.dts b/arch/arm/boot/dts/sun7i-a20-cubieboard2.dts
index 8b1edcd..549d94f 100644
--- a/arch/arm/boot/dts/sun7i-a20-cubieboard2.dts
+++ b/arch/arm/boot/dts/sun7i-a20-cubieboard2.dts
@@ -74,7 +74,11 @@
 
 		i2s0: i2s@01c22000 {
 			pinctrl-names = "default";
-			pinctrl-0 = <&i2s0_pins_a>;
+			/*pinctrl-0 = <&i2s0_pins_a>;*/
+			status = "okay";
+		};
+
+		codec: codec@01c22c00 {
 			status = "okay";
 		};
 
@@ -112,7 +116,7 @@
 			};
 		};
 
-\		gmac: ethernet@01c50000 {
+		gmac: ethernet@01c50000 {
 			pinctrl-names = "default";
 			pinctrl-0 = <&gmac_pins_mii_a>;
 			phy = <&phy1>;
diff --git a/sound/soc/codecs/Makefile b/sound/soc/codecs/Makefile
index 293c215..dc18e5d 100644
--- a/sound/soc/codecs/Makefile
+++ b/sound/soc/codecs/Makefile
@@ -257,6 +257,7 @@ obj-$(CONFIG_SND_SOC_STA350)   += snd-soc-sta350.o
 obj-$(CONFIG_SND_SOC_STA529)   += snd-soc-sta529.o
 obj-$(CONFIG_SND_SOC_STAC9766)	+= snd-soc-stac9766.o
 obj-$(CONFIG_SND_SOC_TAS5086)	+= snd-soc-tas5086.o
+obj-$(CONFIG_SND_SOC_TAS5716)	+= snd-soc-tas5716.o
 obj-$(CONFIG_SND_SOC_TLV320AIC23)	+= snd-soc-tlv320aic23.o
 obj-$(CONFIG_SND_SOC_TLV320AIC23_I2C)	+= snd-soc-tlv320aic23-i2c.o
 obj-$(CONFIG_SND_SOC_TLV320AIC23_SPI)	+= snd-soc-tlv320aic23-spi.o
diff --git a/sound/soc/sunxi/Kconfig b/sound/soc/sunxi/Kconfig
index be99ad4..9f01df6 100644
--- a/sound/soc/sunxi/Kconfig
+++ b/sound/soc/sunxi/Kconfig
@@ -4,7 +4,7 @@ config SND_SUNXI_SOC_CODEC
 	select REGMAP_MMIO
 	default y
 
-config SND_SUNXI_SOC_I2s
+config SND_SUNXI_SOC_I2S
 	tristate "Sunxi I2S support"
 	select SND_SOC_GENERIC_DMAENGINE_PCM
 	select REGMAP_MMIO
